Library/Homebrew/cmd/install.rb

```diff
@@ -324,7 +324,7 @@ def check_cellar
   def symlink_ld_so
     ld_so = HOMEBREW_PREFIX/"lib/ld.so"
     return if ld_so.readable?
-    sys_interpreter = ["/lib64/ld-linux-x86-64.so.2", "/lib/ld-linux.so.3", "/lib/ld-linux.so.2", "/lib/ld-linux-armhf.so.3"].find do |s|
+    sys_interpreter = ["/lib64/ld-linux-x86-64.so.2", "/lib/ld-linux.so.3", "/lib/ld-linux.so.2", "/lib/ld-linux-armhf.so.3", "/system/bin/linker"].find do |s|
       Pathname.new(s).executable?
```

Comment:
You can submit this change to Linuxbrew/brew.

Reply:
Please submit this change to Linuxbrew/brew in a separate PR.

Reply:
https://github.com/Linuxbrew/brew/pull/649